deps.macos/80-carla.zsh

```diff
@@ -0,0 +1,65 @@
+autoload -Uz log_debug log_error log_info log_status log_output
+
+## Dependency Information
+local name='carla'
+local version='2.6.0-alpha1'
+local url='https://github.com/falkTX/Carla.git'
+local hash='31f9a6eef93f205a97deafd251dc4dcb6f5bf5c9'
+
+## Build Steps
+setup() {
+  log_info "Setup (%F{3}${target}%f)"
+  setup_dep ${url} ${hash}
+}
+
+clean() {
+  cd "${dir}"
+
+  if [[ ${clean_build} -gt 0 && -d "build_${arch}" ]] {
+    log_info "Clean build directory (%F{3}${target}%f)"
+
+    rm -rf "build_${arch}"
+  }
+}
+
+config() {
+  autoload -Uz mkcd progress
+
+  log_info "Config (%F{3}${target}%f)"
+
+  args=(
+    ${cmake_flags}
+    -DCARLA_BUILD_FRAMEWORKS=TRUE
```

Comment:
If possible add `:BOOL` to the cache variable declaration here, and prefer `ON|OFF` for declarations from CLI.